client/src/views/Create.vue

```diff
@@ -28,137 +30,36 @@
 
 <script>
 // @ is an alias to /src
-import Debug from 'debug';
 import FormWrapper from '@/components/FormWrapper.vue';
-import CosmologyForm from '@/components/CosmologyForm.vue';
-import TransferForm from '@/components/TransferForm.vue';
-import FilterForm from '@/components/FilterForm.vue';
 import HaloExclusion from '@/components/HaloExclusion.vue';
-import HaloProfileForm from '@/components/HaloProfileForm.vue';
-import HaloModelForm from '@/components/HaloModelForm.vue';
-import constants from '@/constants/backend_constants';
-import baseURL from '@/env';
-
-const debug = Debug('Create.vue');
+import Blank from '@/constants/initial_state.json';
```

Comment:
`Blank` seems like a vague variable name. Can we use something like `initialState`? Plus it is an object and not a class so the capital letter seems odd.

Reply:
Fixed in recent commit